presto-docs/src/main/sphinx/release/release-0.220.rst

```diff
@@ -0,0 +1,55 @@
+=============
+Release 0.219
+=============
+
+General Changes
+---------------
+* Fix an issue when executing spatial join on top of a right join. See :issue:`12672`.
+* Fix an issue where ``TRY`` cannot catch the failures when ``ROW`` and ``ARRAY`` with ``NULL`` elements fail to compare.
```

Comment:
```
* Fix an issue where ``TRY`` would not catch failures when a ``ROW`` or ``ARRAY`` with ``NULL`` elements failed to compare.
```

I'm actually not sure what "failed to compare" means.  What's it comparing with?

Reply:
```
Fix an issue where ``TRY`` cannot catch the failures when 2 ``ROWs`` or 2 ``ARRAYs``
with ``NULL`` elements fail to compare.
```